uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/request-ctrl.js

```diff
@@ -789,7 +789,16 @@ define([
           this.error = errors.error_message;
         else {
           if (_.isObject(errors)) {
-            this.error = JSON.stringify(errors)
+            var errorMessages = [];
+
+            _.each(errors, function (value, key) {
+              var errorMessage = _.isArray(value) ? value.join(', ') : JSON.stringify(value);
+
+              errorMessage = errorMessage + ' for key ' + key;
+              errorMessages.push(errorMessage);
+            });
+
+            this.error = errorMessages.join(', ');
```

Comment:
I don't know how many different formats error can have and that was the reason for having a discussion on standardization of error handling. I have seen this and the use of error_message so far.